File: project/sb-bets/models/Bet.go
package models

// Bet comment
type Bet struct {
	ID              int64  `json:"id"`
	UserID          int64  `json:"userID"`
	UserName        string `json:"userName"`
	Value           string `json:"value"`
	IsPayed         bool   `json:"isPayed"`
	BetOnIndex      int    `json:"betOnIndex"`
	LineID          int64  `json:"lineID"`
	LineType        int    `json:"lineType"`
	LineDescription string `json:"lineDescription"`
	GameID          int64  `json:"gameID"`
	GameName        string `json:"gameName"`
}

File: project/sb-client/src/models/Bet.ts
export interface Bet {
    id: number,
    userID: number,
    userName: string,
    value: string,
    isPayed: boolean,
    betOnIndex: number,
    lineID: number,
    lineType: number,
    lineDescription: string,
    gameID: number,
    gameName: string,
}